src/ui/agents/KivaAgent.ts

```diff
@@ -107,7 +106,8 @@ export default class KivaAgent implements Agent {
     }
 
     captureAndSendError(error: any, message: string) {
-        const msg: string = I18n.getKey(message);
+        const errorDetails = (" (" + error.response.data.code + ": " + error.response.data.message + ")").toString();
```

Comment:
nit: You could use here this type of string substitution: `${error.response.data.code}: ${error.response.data.message}`